### PR TITLE
Release 2.0.0

### PR DESCRIPTION
**Pull request by orakley**

### CODE REVIEW

**Review by orakley — COMMENTED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -64,69 +64,130 @@ const config = {
...
+            "isGroup": true,
+            "children": [{
+                "index": 0,
+                "value": "",
+                "operator": "contain",
+                "level": 1,
+                "uuid": "857f1a30-a133-4293-99f0-7a35121926a0",
+                "type": {
+                    "name": "Hausnummer",
+                    "identificator": "housenumber",
```
Comment (oliverborner) [inline README.md]: identifier

**Review by oliverborner — COMMENTED** [review]
LGTM!

**Review by oliverborner — APPROVED** [review]
LGTM!

**Pull request merged**